frontend/wallet/src/render/components/Link.re

```diff
@@ -1,3 +1,8 @@
+type kind =
+  | Blue
+  | Grey
+  | Red;
+
```

Comment:
Refactored the link to have three different styles, with default to Blue. Herold is taking another pass at these styles so they'll likely change.

Reply:
Any thoughts on using polymorphic variants for these kinds of color things